packages/documentation/pages/usage/components/filters.vue

```diff
@@ -167,12 +199,13 @@ export default defineComponent({
 							Kotti.Filters.Operation.Float.LESS_THAN_OR_EQUAL,
 							Kotti.Filters.Operation.Float.IS_EMPTY,
 						],
-						prefix: settings.value.prefix,
-						step: 0.001,
-						suffix: settings.value.suffix,
+						prefix: settings.value.numberPrefix ?? undefined,
```

Comment:
Most of these can probably stay `null`. The `undefined` only comes into play when you’re potentially passing these props to components that shouldn’t get them.

I’m fine with keeping this as-is though as it’s not technically wrong